src/main/java/com/faforever/api/config/elide/checks/IsUser.java

```diff
@@ -0,0 +1,14 @@
+package com.faforever.api.config.elide.checks;
+
+import com.yahoo.elide.security.User;
+import com.yahoo.elide.security.checks.UserCheck;
+
+/**
+ * @author Dragonfire
+ */
+public class IsUser extends UserCheck {
+  @Override
+  public boolean ok(User user) {
+    return user != null & user.getOpaqueUser() != null;
+  }
+}
```

Comment:
The structure of this class is not consistent with the existing classes and as recommended by Elide: http://elide.io/pages/guide/03-security.html